ibm/mas_devops/roles/suite_config/tasks/main.yml

```diff
@@ -48,3 +73,13 @@
     state: present
     definition: "{{ lookup('template', item.path) }}"
   with_items: "{{ mas_config_files.files }}"
+
```

Comment:
If want to have a role create a workspace let's put it in a seperate new role please, e.g. `mas_workspace_create` ... this role's job is to apply configurations from a directory.  If one wants to use this to create a workspace they create the yaml for the workspace and pass the directory containing that file into the role (as we do today in daily FVT).

Reply:
Created a new role suite_config_workspace and added the role in install_suite.yml . Please let me know if this is correct .
@santoshjpawar - FYI